### PR TITLE
change some chinese expression

### PR DESCRIPTION
**Pull request by liwjGhostcloud**
change some chinese expression. when a chinese read more clear to understood the docs`s expression

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5914)

### CODE REVIEW

**Comment by liwjGhostcloud** [comment]
add cla/linuxfoundation

File: cn/docs/tasks/administer-cluster/cpu-memory-limit.md [hunk 1]
```diff
@@ -14,9 +14,8 @@ redirect_from:
 
 
 
-默认情况下，Pod 运行没有限制 CPU 和内存。
-这意味着系统中的任何 Pod 将能够像执行该 Pod 所在的节点一样，使用足够多 CPU 和内存。
-
+默认情况下，Pod 运行没有限制 CPU使用量 和内存使用量。
```
Comment (zjj2wry) [inline cn/docs/tasks/administer-cluster/cpu-memory-limit.md]: 这里多了一个空格

Comment (liwjGhostcloud) [reply cn/docs/tasks/administer-cluster/cpu-memory-limit.md]: @zjj2wry 已经将空格删除

**Comment by zjj2wry** [comment]
@liwjGhostcloud please squash your commits, then will lgtm, thanks

**Pull request merged**